GOVERNANCE.md

```diff
@@ -106,6 +105,12 @@ The current team members are:
 - Will Browne ([Grafana Labs](https://grafana.com/))
 - Zoltán Bedi ([Grafana Labs](https://grafana.com/))
 
+### Hall of fame 🕺🏽🌟
```

Comment:
```suggestion
### Emeritus Members🕺🏽🌟
```

Many other projects use Emeritus for this.

```
What Is an Emeritus Core Maintainer?
In academia, when well-regarded faculty move on to other interests, the school may give them the title emeritus professor. This title confers honor on the recipient. In this way, the professor remains associated with the department and school in which she or he taught. But an emeritus professor is no longer required to teach courses, fill research obligations, or serve on committees.
```

Reply:
I really like this suggestion!

Reply:
Same, think this makes a lot of sense